integreat_cms/cms/templates/feedback/_feedback_widget.html

```diff
@@ -25,22 +25,31 @@
                     {% if feedback.object_url %}
                         <a href="{{ feedback.object_url }}">{{ feedback.object_name }}</a>
                     {% else %}
-                        {{ feedback.object_name }}
+                        <div class="feedback-entry">
+                            <span class="feedback-entry-content overflow-hidden overflow-ellipsis whitespace-nowrap">
```

Comment:
```suggestion
                            <span class="feedback-entry-content">
```
I guess these classes can be removed because they are set in the css stylesheet